src/mame/galaxian/galaxian.cpp

```diff
@@ -710,6 +710,52 @@ t) 'bagmanmc'
   - DSW bit 6 was previously used for "Bonus Lives" settings, but it has no effect
     in this set because of 'NOP' instructions from 0x3501 to 0x3507.
 
+u) uniwars
+
+    Uniwars Warp Analysis on an Irem PCB from PrSwan (Paul Swan)
+    ------------------------------------------------------------
+
+    - What is the warp effect?
+      The game PCB has several fly wires of which Red & Green relate to the warp effect.
```

Comment:
This doesn’t actually answer the question of what the warp effect is.  Is it a sound effect?  A video effect?  Something else?